.buildkite/pipeline.build.yml

```diff
@@ -216,9 +216,11 @@
   instance_size: large
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
-    # Todo (krfricke): Move mosaicml to train-test-requirements.txt
-    - pip install "mosaicml==0.12.1"
     - DOC_TESTING=1 ./ci/env/install-dependencies.sh
+    # Todo (krfricke): Move mosaicml to train-test-requirements.txt
+    # TODO(scottjlee): Move datasets to train/data-test-requirements.txt 
+    # (see https://github.com/ray-project/ray/pull/38432/)
+    - pip install "datasets==2.14.0" "protobuf==3.20.1" "mosaicml==0.15.0"
```

Comment:
do we still get the protobuf issue if mosaicml is moved to before `install-dependencies.sh`?

Reply:
you mean something like this right? i can give it a try.
```
- pip install mosaicml==0.15.0"
- DOC_TESTING=1 ./ci/env/install-dependencies.sh
- pip install "datasets==2.14.0"
```